debian/changelog

```diff
@@ -1,3 +1,19 @@
+folio-okapi (3.1.0-1) xenial; urgency=medium
+
+  * Imported Upstream version 3.1.0
```

Comment:
This line should be removed. Importing upstream versions should go via the https://github.com/folio-org/okapi-debian/tree/debian-upstream branch.